data/sql/updates/pending_db_world/rev_1586619822318777300.sql

```diff
@@ -154,4 +167,52 @@ INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `p
 
-UPDATE `creature_template_addon` SET `auras` = '47543' WHERE `entry` IN (26800, 26802, 26805, 27949);
+DELETE FROM `creature_addon` WHERE `guid` IN (116017,116016,115899,115933,115900,115965,115952);
+INSERT INTO `creature_addon` (`guid`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES
+(115899, 1, 33, 1, '47543'),
+(115900, 1, 33, 1, '47543'),
+(115933, 1, 376, 1, '47543'),
+(115952, 1, 376, 1, '47543'),
+(115965, 1, 376, 1, '47543'),
+(116016, 1, 375, 1, '47543'),
+(116017, 1, 375, 1, '47543');
+
+DELETE FROM `creature_template_addon` WHERE `entry` IN (26800,26802,26805,27949);
+INSERT INTO `creature_template_addon` (`entry`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES
+(26800, 1, 333, 1, '47543'),
+(26802, 1, 376, 1, '47543'),
+(26805, 1, 375, 1, '47543'),
+(27949, 1, 333, 1, '47543');
+
+-- The Singing Grove
+DELETE FROM `creature` WHERE `guid` IN (126600,126601,126602,126603,126604,126605,126606,126607,126608,126609,1266090,126440,126441,126442,126443,126444,126445,126446,126447);
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES 
+(126600, 26782, 576, 0, 0, 3, 1, 27469, 0, 356.128, -179.167, -14.2759, 3.80705, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126601, 26782, 576, 0, 0, 3, 1, 27469, 0, 383.348, -184.97, -14.0888, 2.91584, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126602, 26782, 576, 0, 0, 3, 1, 27469, 0, 387.443, -184.48, -14.0888, 2.63267, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126603, 26782, 576, 0, 0, 3, 1, 27469, 0, 458.056, -177.292, -14.0888, 5.79238, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126604, 26782, 576, 0, 0, 3, 1, 27469, 0, 453.272, -178.744, -14.0888, 5.79238, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126605, 26782, 576, 0, 0, 3, 1, 27469, 0, 247.919, -235.086, -8.48918, 2.34172, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126606, 26782, 576, 0, 0, 3, 1, 27469, 0, 251.404, -238.64, -8.25408, 2.34654, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126607, 26782, 576, 0, 0, 3, 1, 27469, 0, 403.166, -313.62, -14.0888, 0.612719, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126608, 26782, 576, 0, 0, 3, 1, 27469, 0, 397.014, -322.266, -14.0888, 0.174134, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126609, 26782, 576, 0, 0, 3, 1, 27469, 0, 431.531, -273.502, -14.0888, 2.62032, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(1266090, 26782, 576, 0, 0, 3, 1, 27469, 0, 431.531, -273.502, -14.0888, 2.62032, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126440, 28231, 576, 0, 0, 3, 1, 27609, 1, 356.727, -174.203, -14.0888, 3.80708, 3600, 0, 0, 37164, 3231, 2, 0, 0, 0, '', 0),
+(126441, 28231, 576, 0, 0, 3, 1, 27609, 1, 383.407, -184.995, -14.0888, 2.74376, 3600, 0, 0, 37164, 3231, 2, 0, 0, 0, '', 0),
```

Comment:
sql```
(126441, 28231, 576, 0, 0, 3, 1, 27609, 1, 383.407, -184.995, -14.0888, 2.74376, 3600, 5, 0, 37164, 3231, 1, 0, 0, 0, '', 0),
```

Reply:
No, that is wrong. It should not have random movement